README.md

```diff
@@ -67,57 +63,61 @@ The background services can be stopped as a group with:
-
-Once paired, your unit will use Mycroft API keys for services such as Speech-to-Text (STT), weather and various other skills.
-
 ### Skills
 
-Mycroft is nothing without skills.  There are a handful of default skills that are downloaded automatically to your `/opt/mycroft/skills` directory, but most need to be installed explicitly.  See the [Skill Repo](https://github.com/MycroftAI/mycroft-skills#welcome) to discover skills made by others.  Please share your own interesting work!
+Mycroft is nothing without skills.  There are a handful of default skills, but most need to be installed explicitly.  
+See the [Skill Repo](https://github.com/MycroftAI/mycroft-skills#welcome) to discover skills made by others.  
+Please share your own interesting work!
 
 ## Behind the scenes
 
-### Pairing Information
-Pairing information generated by registering with Home is stored in:
+### Identity Information
+A unique uuid and pairing information generated by registering with Home is stored in:
 `~/.config/mycroft/identity/identity2.json` <b><-- DO NOT SHARE THIS WITH OTHERS!</b>
 
+This file uniquely identifies your device and should be kept safe
+
 ### Configuration
 Mycroft's configuration consists of 4 possible locations:
 - `mycroft-core/mycroft/configuration/mycroft.conf`(Defaults)
-- [Mycroft Home](https://home.mycroft.ai) (Remote)
+- [Mycroft Home](https://home.mycroft.ai) (Remote) 
 - `/etc/mycroft/mycroft.conf` (Machine)
 - `$XDG_CONFIG_DIR/mycroft/mycroft.conf` (which is by default `$HOME/.config/mycroft/mycroft.conf`) (USER)
 
 When the configuration loader starts, it looks in these locations in this order, and loads ALL configurations. Keys that exist in multiple configuration files will be overridden by the last file to contain the value. This process results in a minimal amount being written for a specific device and user, without modifying default distribution files.
 
-### Using Mycroft Without Home
+Remote configuration only exists if you explicitly enabled a backend
+
+### *Home* Device and Account Manager
+Mycroft AI, Inc. maintains a device and account management system known as Mycroft Home. Developers may sign up at: https://home.mycroft.ai
 
-If you do not wish to use the Mycroft Home service, before starting Mycroft for the first time, create `$HOME/.config/mycroft/mycroft.conf` with the following contents:
+By default, mycroft-core is configured to not use Home. 
 
-```
-{
-  "skills": {
-    "blacklisted_skills": [
-      "mycroft-configuration.mycroftai",
-      "mycroft-pairing.mycroftai"
-    ]
-  }
-}
-```
+When enabled by saying "Hey Mycroft, pair my device" (or any other request verbal request) you will be informed that your device needs to be paired. 
+Mycroft will speak a 6-digit code which you can enter into the pairing page within the [Mycroft Home site](https://home.mycroft.ai).
 
-### API Key Services
+The Mycroft backend provides access to a range of API keys for specific services. 
```

Comment:
I might reword slightly since they provide a proxy for using these APIs in a specific way, not keys for using the APIs directly:
```
The Mycroft backend provides access to a range of third-party APIs for specific services.
```